docs/application/tizen-studio/release-notes/release-notes.md

```diff
@@ -1 +1 @@
-4-6-release-notes.md
\ No newline at end of file
+5-0-release-notes.md
```

Comment:
Would you please check the OS system that you modified this file?
It seems "release-notes.md" is going to `'5-0-release-notes.md'$'\n'`.

```
...tizen-docs/docs/application/tizen-studio/release-notes(test_tizen_studio_5.0)$ ls -l release-notes.md
lrwxrwxrwx 1 ... 21 10월 24 09:53 release-notes.md -> '5-0-release-notes.md'$'\n'
```

Invalid soft link caused error on docs.tizen.org site before,
so we fixed the error as like https://github.com/Samsung/tizen-docs/pull/1619.
I'm worrying about reproducing of the same error.

Reply:
@vivek-ellur Please check to the comment above.

Reply:
Hello, @vivek-ellur 
Would you please check my comment? It is to prevent the malfunction of the docs.tizen.org.

Reply:
@mijin85cho I updated the file in windows. Could you please let me know, If any changes need to be done.

Reply:
@vivek-ellur Is it possible to update the release-notes.md file with Linux? 
The different OS system use different way to handle the space/tab/new line,
and `$'\n'` caused by windows in the symbolic link didn't work well before.

Reply:
Hello @vivek-ellur Would you please check the comment?
As we are to release Tizen Studio 5.0 on 10.31, this PR should be merged till that day.

For now https://docs1.stg.tizen.org/staging/1701/ is showing some errors.

For example : 
 https://docs1.stg.tizen.org/staging/1701/application/tizen-studio/release-notes/5-0-release-notes/ exist, but
 https://docs1.stg.tizen.org/staging/1701/application/tizen-studio/release-notes/release-notes/ doesn't exist.

You can test normal operating with 
https://docs.tizen.org/application/tizen-studio/release-notes/4-6-release-notes/ and
https://docs.tizen.org/application/tizen-studio/release-notes/release-notes/

Reply:
@mijin85cho can I close this and raise new request?

Reply:
@vivek-ellur Yes you can close this PR and raise a new PR. Please mark in the new PR that it is to wrap up this PR.

I recommend the reviewer of this PR continue reviewing the new PR that will be raised